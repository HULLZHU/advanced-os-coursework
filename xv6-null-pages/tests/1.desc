ensure program is mapped somewhere other than at page 0
