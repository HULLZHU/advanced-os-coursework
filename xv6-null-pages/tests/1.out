XV6_TEST_OUTPUT 1
